hoomd/md/pair/pair.py

```diff
@@ -268,6 +268,7 @@ class LJ(Pair):
         default_r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
         default_r_on (float): Default turn-on radius :math:`[\mathrm{length}]`.
         mode (str): Energy shifting/smoothing mode.
+        tail_correction (bool): Whether to apply a tail correction.
 
     `LJ` specifies that a Lennard-Jones pair potential should be
     applied between every non-excluded particle pair in the simulation.
```

Comment:
Add documentation that defines the functional form of the tail correction in LaTeX math and cite the source of the tail correction.

Reply:
See [this commit](https://github.com/glotzerlab/hoomd-blue/pull/1138/commits/e2fbdd4041d7d4f2d8359823e5688692762fc8f7)